deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-seka/deployment.yaml

```diff
@@ -20,10 +20,10 @@ spec:
               mountPath: /data
           resources:
             limits:
-              cpu: "28"
+              cpu: "7"
               memory: 58Gi
```

Comment:
I am curious why memory is not being reduced?

Reply:
Only because others with the same CPU reduction are still at 58Gi.